src/main/java/uk/gov/hmcts/reform/sscs/job/ProcessingVenueMigrationJob.java

```diff
@@ -56,8 +81,18 @@ boolean isRollback() {
     }
 
     public void updateCaseData(SscsCaseData caseData, String venue) {
+        String postCode = resolvePostCode(caseData);
+        RegionalProcessingCenter newRpc = regionalProcessingCenterService.getByPostcode(postCode);
+        String venueEpimsId = venueService.getEpimsIdForVenue(venue);
+        CourtVenue courtVenue = refDataService.getCourtVenueRefDataByEpimsId(venueEpimsId);
+        log.info("Setting processing venue to ({}), RPC to ({}), and Case management location to region ({})",
+            venue, newRpc.getName(), courtVenue.getRegionId());
+        caseData.setCaseManagementLocation(CaseManagementLocation.builder()
+            .baseLocation(newRpc.getEpimsId())
+            .region(courtVenue.getRegionId()).build());
+        caseData.setRegionalProcessingCenter(newRpc);
+        caseData.setRegion(newRpc.getName());
         caseData.setProcessingVenue(venue);
```

Comment:
are we taking the benefitType into account when migrating the venue, because the new venue seems to be different for different benefit types, for some of the locations